PVEntityGenerator/PVEntityGenerator.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26430.14
+# Visual Studio Version 17
+VisualStudioVersion = 17.2.32526.322
```

Comment:
it should not be required to update the dependency here in this PR - if required this should go to a separate PR.